OrchardCore.sln

```diff
@@ -216,6 +216,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "build", "build", "{184139CF
 	ProjectSection(SolutionItems) = preProject
 		src\OrchardCore.Build\Dependencies.AspNetCore.props = src\OrchardCore.Build\Dependencies.AspNetCore.props
 		src\OrchardCore.Build\Dependencies.props = src\OrchardCore.Build\Dependencies.props
+		Directory.Build.props = Directory.Build.props
```

Comment:
You could hosted in the `OrchareCore.Build` folder instead

Reply:
That's `src/OrchardCore.Build` but this file also applies to files in `test`, so that doesn't feel like the right level to me.

Reply:
`Dependencies.prop` for instance has been used in test project too

Reply:
The convention for Directory.Build.props is that it applies to subfolders. The probing stops at the first round file, hence the other changes here that chain the files to one another when multiple levels exist.

This way, all future projects added will pick this property up by default.

Reply:
https://learn.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2022#directorybuildprops-and-directorybuildtargets